src/testlib/include/testlib/helpers.hpp

```diff
@@ -40,6 +43,21 @@ inline std::string toString(const YAML::Node& node) {
     return std::string{out.c_str()};
 }
 
+inline std::string toString(int i) {
+    return std::to_string(i);
+}
+
+// TODO: this is duped with MongoTestFixture
+inline void dropAllDatabases(mongocxx::pool::entry& client) {
```

Comment:
Could we have the fixture's function just call this?

Reply:
Good call - took a bit of template magic because they're of different types but it ended up working.